### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by ghost** [kyleneideck/BackgroundMusic#706] [Q&A]
I'm not good at this coding stuff, but I tried to install the code but it gave me an error. I have Xcode installed btw. The code said to report it and to include the log.
[build_and_install.log](https://github.com/kyleneideck/BackgroundMusic/files/13631529/build_and_install.log)

**Comment by LoackyBit** [comment]
@BobANDJoe don't build the code by your own, instead use the precompiled version: [BackgroundMusic-0.5.0.pkg](https://github.com/kyleneideck/BackgroundMusic/releases/download/v0.5.0/BackgroundMusic-0.5.0.pkg)

**Comment by LoackyBit** [comment]
@kyleneideck can be marked as answered
